kentikapi/users_integration_test.go

```diff
@@ -598,6 +602,366 @@ func TestClient_DeleteUser(t *testing.T) {
 	}
 }
 
+// go test -tags kentikapi -run TestClient_CreateUser_Retry -count=1 -v ./... | tee test.log
+
+func TestClient_GetUser_Retry(t *testing.T) {
+	tests := []struct {
+		name           string
+		retryMax         *int
+		retryStatusCodes []int
+		retryMethods     []string
+		responses	   []httpResponse
+		expectedResult *models.User
+		expectedError  bool
+	}{
+		{
+			name:          "retry until invalid response format",
+			responses: []httpResponse{
+				newErrorHTTPResponse(http.StatusBadGateway),
+				newErrorHTTPResponse(http.StatusBadGateway),
+				{http.StatusOK, "invalid JSON"},
+			},
+			expectedError: true,
+		},
+		{
+			name:
```

Comment:
```suggestion
			name: "status...",
```
IMO redundant line break.
The same with _responses_ field below